src/processors/data-processor.js

```diff
@@ -100,17 +100,39 @@ export const PARSE_FIELD_VALUE_FROM_STRING = {
  *  options: {centerMap: true, readOnly: true}
  * }));
  */
-export function processCsvData(rawData) {
-  // here we assume the csv file that people uploaded will have first row
-  // as name of the column
-  // TODO: add a alert at upload csv to remind define first row
-  const result = csvParseRows(rawData);
-  if (!Array.isArray(result) || result.length < 2) {
-    // looks like an empty file, throw error to be catch
-    throw new Error('Read File Failed: CSV is empty');
+export function processCsvData(rawData, header) {
```

Comment:
This seems to duplicate the CSVLoader with header detection etc. Can we unify?

Is this unique to CSV data? not other tables? Rename the functions?